src/sentry_plugins/pivotal/plugin.py

```diff
@@ -0,0 +1,149 @@
+"""
+pivotal.plugin
+~~~~~~~~~~~~~
+
+:copyright: (c) 2016 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from rest_framework.response import Response
+from django import forms
+from django.conf.urls import url
+from django.utils.translation import ugettext_lazy as _
+from django.utils.encoding import force_text
+from sentry.plugins.bases.issue2 import IssuePlugin2, IssueGroupActionEndpoint
+from sentry.utils.http import absolute_uri
+
+import sentry_plugins
+import pyvotal
+
+
+class PivotalPlugin(IssuePlugin2):
+    author = 'Sentry Team'
+    author_url = 'https://github.com/getsentry/sentry'
+    version = sentry_plugins.VERSION
+    description = "Integrate Pivotal Tracker stories by linking a project and account."
+    resource_links = [
+        ('Bug Tracker', 'https://github.com/getsentry/sentry-plugins/issues'),
+        ('Source', 'https://github.com/getsentry/sentry-plugins'),
+    ]
+
+    slug = 'pivotal'
+    title = 'Pivotal Tracker'
+    conf_title = title
+    conf_key = 'pivotal'
+
+    def get_group_urls(self):
+        _patterns = super(PivotalPlugin, self).get_group_urls()
+        _patterns.append(url(r'^autocomplete',
+                             IssueGroupActionEndpoint.as_view(view_method_name='view_autocomplete',
+                                                              plugin=self)))
+        return _patterns
+
+    def is_configured(self, request, project, **kwargs):
+        return all(self.get_option(k, project) for k in ('token', 'project'))
+
+    def get_api_client(self, project):
+        return pyvotal.PTracker(token=self.get_option('token', project))
+
+    def get_link_existing_issue_fields(self, request, group, event, **kwargs):
+        return [{
+            'name': 'issue_id',
+            'label': 'Story',
+            'default': '',
+            'type': 'select',
+            'has_autocomplete': True,
+            'help': ('Search Pivotal Stories by name or description.')
+        }, {
+            'name': 'comment',
+            'label': 'Comment',
+            'default': absolute_uri(group.get_absolute_url()),
```

Comment:
This `absolute_uri` is redundant here since `group.get_absolute_url()` is doing that.